pwndbg/commands/radare2.py

```diff
@@ -25,8 +25,11 @@ def r2(arguments, no_seek=False):
 
     # Build up the command line to run
     cmd = ['radare2', filename]
+    addr = pwndbg.regs.pc
+    if pwndbg.wrappers.checksec.pie_status() == "PIE enabled":
```

Comment:
```suggestion
    if pwndbg.elf.is_pie():
```

Can you check if that works fine?

Reply:
Exception occured: r2: 'module' object has no attribute 'is_pie' (<type 'exceptions.AttributeError'>)

elf is a module. 

Maybe we need an object?

Reply:
Oh it should be:
```python
pwndbg.elf.get_elf_info(pwndbg.proc.exe).is_pie
```

Reply:
Checked.  But that line should be:
```
if pwndbg.elf.get_elf_info(filename).is_pie:
```

Reply:
Sure, can you change it to this? :)

Reply:
Can you see it ?

Reply:
Yeah, thx!